src/utils/constants.ts

```diff
@@ -22,6 +22,21 @@ export enum PartnerAccessCodeStatusEnum {
   CODE_EXPIRED = 'CODE_EXPIRED',
 }
 
+const getPartnerDetails = (userObject: UserEntity) => {
+  const object = userObject.partnerAccess
+    ? userObject.partnerAccess.partner
+    : userObject.partnerAdmin.partner;
+
+  return {
+    id: object.id,
+    createdAt: object.createdAt,
+    updatedAt: object.updatedAt,
```

Comment:
can we not send the `createdAt` and `updatedAt` for partner? i think its TMO for users